chaoslib/litmus/container_kill/containerd_chaos/crictl-chaos.yml

```diff
@@ -1,5 +1,32 @@
 ---
 - block:
+    - block: 
+
+      - name: Select the app pod
+        shell: >
+          kubectl get pod -l {{ a_label }} -n {{ a_ns }}
+          -o=custom-columns=:metadata.name --no-headers
+          | shuf | head -1 
+        args:
+          executable: /bin/bash
+        register: app_pod_name
+
+      - name: Record app pod name
+        set_fact:
+          app_pod: "{{ app_pod_name.stdout }}"
+
+      when: "app_pod is not defined or app_pod == ''"
+
+    - name: Identify the application node
+      shell: >
+        kubectl get pod {{ app_pod }} -n {{ a_ns }}
+        --no-headers -o custom-columns=:spec.nodeName
```

Comment:
We can directly get the node name using `kubectl get pod -l  {{ a_label }} -n {{ a_ns }}
        --no-headers -o custom-columns=:spec.nodeName` command so why are we adding two extra steps. First get the app-pod name and then get the node name.

Reply:
Because label could be same for many pod.